src/components/addBook.js

```diff
@@ -0,0 +1,19 @@
+const AddBook = () => (
+  <div className="Form">
+    <h3>ADD NEW BOOK</h3>
+    <form className="FormInput">
+      <input type="text" placeholder="Book title" value="" className="book-title" />
```

Comment:
- [ ] An author input is also needed, please add it to comply with the project requirements.